make/lint.mk

```diff
@@ -0,0 +1,16 @@
+.PHONY: lint
+## Runs linters on Go code files and YAML files
+lint: lint-go-code lint-yaml
+
+.PHONY: lint-yaml
+## runs yamllint on all yaml files
+lint-yaml:
+	$(Q)yamllint -c .yamllint ./
```

Comment:
Did you intend to include the .yamllint file in this PR or a diff one?

Reply:
Nevermind, just saw https://github.com/kubesaw/ksctl/pull/2 🙂

Reply:
see https://github.com/kubesaw/ksctl/pull/2

Reply:
funny, GH page didn't show me your responses, that's why I replied. Bloody GH page caching